plugins/jukealert-paper/src/main/java/com/untamedears/jukealert/model/actions/impl/KillPlayerAction.java

```diff
@@ -8,33 +8,32 @@
 import org.bukkit.inventory.ItemStack;
 import vg.civcraft.mc.civmodcore.inventory.gui.DecorationStack;
 import vg.civcraft.mc.civmodcore.inventory.gui.IClickable;
-import vg.civcraft.mc.civmodcore.inventory.items.ItemUtils;
 import vg.civcraft.mc.namelayer.NameAPI;
 
 public class KillPlayerAction extends LoggablePlayerVictimAction {
-	
+
 	public static final String ID = "KILL_PLAYER";
 
 	public KillPlayerAction(long time, Snitch snitch, UUID player, Location location, UUID victim) {
 		super(time, snitch, player, location, victim.toString());
 	}
-	
+
 	public String getVictimName() {
 		return NameAPI.getCurrentName(getVictimUUID());
 	}
-	
+
 	public UUID getVictimUUID() {
 		return UUID.fromString(victim);
 	}
 
 	@Override
 	public IClickable getGUIRepresentation() {
 		ItemStack is = new ItemStack(Material.DIAMOND_SWORD);
-		ItemUtils.addGlow(is);
+		is.getItemMeta().setEnchantmentGlintOverride(true);
```

Comment:
Not sure these work, iirc `.getItemMeta()` creates a duplicate meta, so any changes to it would then need to be saved back onto the item.